website/docs/d/serverless_instance.html.markdown

```diff
@@ -11,7 +11,6 @@ Provides a Serverless Instance.
 `mongodbatlas_serverless_instance` describe a single serverless instance. This represents a single serverless instance that have been created.
 > **NOTE:**  Serverless instances are in a preview release and do not support some Atlas features at this time.
 For a full list of unsupported features, see [Serverless Instance Limitations](https://docs.atlas.mongodb.com/reference/serverless-instance-limitations/).
```

Comment:
We'll need to update this line for both data sources and the resource.  
```suggestion
> **NOTE:**  Serverless instances do not support some Atlas features at this time.  For a full list of unsupported features see [Serverless Instance Limitations](https://docs.atlas.mongodb.com/reference/serverless-instance-limitations/).
```

Reply:
Thanks for catching that, I totally missed the labeling as in preview release